includes/class.llms.install.php

```diff
@@ -613,6 +615,24 @@ public static function install() {
 
 	}
 
+	/**
+	 * Retrieve permalinks structure to verify if they are set, and any new defaults are saved
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public static function verify_permalinks() {
+		if ( ! get_option( 'lifterlms_permalinks' ) ) {
```

Comment:
Perhaps this should always be run? If any add-ons have new defaults to set, they should be added in the site language immediately.

Reply:
Switching to having add-ons handle their own saved option for the slug(s) they're changing, to avoid race conditions and decrease complexity.